### ISSUE TITLE
Accessibility Issue: Main nav hamburger menu lacks focus state color change when tabbing through page

### ISSUE THREAD

**Issue by trubach** [web-illinois/ilw-header-menu#23]
The hamburger menu button on tablet / mobile devices is currently lacking a color change when it gets focus while tabbing through a page.

Below is two screen captures of the behavior. The first is the page title having tab focus, right before the menu does. The second is the page with the hamburger menu having focus, which doesn't change. The hamburger button should match the focus state specified in the design and pattern library.

Page title having tab focus:

<img width="793" height="234" alt="Image" src="https://github.com/user-attachments/assets/0dd7f7c0-2b24-4239-b49b-d91e9570afac" />

Hamburger menu having focus:

<img width="817" height="227" alt="Image" src="https://github.com/user-attachments/assets/a1a59c64-7d66-4b7b-90d6-a37a49af4154" />

This issue was tested on https://builder3.toolkit.illinois.edu/component/ilw-header-menu/1-2/index.html and reported by Tim Offenstein (timo@illinois.edu)

**Comment by marvel-uiuc** [comment]
We will check with design to see if the normal button focus styling should be used.

**Comment by trubach** [comment]
This is a duplicate issue to https://github.com/web-illinois/ilw-header/issues/10 - closing this since it's being worked on over there
